app/dlc-suredbits-client/src/main/scala/org/bitcoins/sbclient/SbClient.scala

```diff
@@ -117,8 +140,7 @@ object SbClient {
       tradingPair: TradingPair,
       requestType: RequestType,
       eclairApi: EclairApi,
-      endpoint: String = "https://test.api.suredbits.com/dlc/v0")(
-      implicit system: ActorSystem): Future[String] = {
+      endpoint: String)(implicit system: ActorSystem): Future[String] = {
```

Comment:
I think that it makes sense to get rid of `endpoint` here and use the class one as it would be weird to have an `SbClient(endpoint1).requestAndPay(endpoint2)`

Reply:
This is in the `SbClient` `object`, I think it makes sense to keep it there and have the ability to pass in an endpoint, this makes the tests much cleaner and it can be reused for other things in the future.

Reply:
ah didn't realize it was in the object